web/template/link_person.gohtml

```diff
@@ -73,13 +73,15 @@
         {{ end }}
 
         <div class="govuk-button-group">
-          <button class="govuk-button" data-module="govuk-button" type="submit">
             {{ if .OtherPerson.ID }}
-              Link records
+              <button class="govuk-button" data-module="govuk-button app-loading-button" data-loading-button-message="#link-records-message" type="submit">
```

Comment:
A couple of oddities here: `app-loading-button` needs to overwrite `govuk-button`, and `data-loading-button-message` is only needed if you've got a custom message to show the user which we don't in this case
```suggestion
              <button class="govuk-button" data-module="app-loading-button" type="submit">
```